tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -117,7 +117,7 @@ def get_performance_hint(device) -> properties.hint.PerformanceMode:
                         logger.warning(f"No device {device} performance hint is set.")
                         perf_hint = properties.hint.PerformanceMode.UNDEFINED
                     else:
-                        perf_hint = properties.hint.PerformanceMode(args.perf_hint.upper())
+                        perf_hint = args.perf_hint.upper()
```

Comment:
python3.7 seems construct the enum from a string is not supported in pybind. I'm going to try a higher version.

Reply:
@jiwaszki Hi, Jan, can you confirm this? currently in my env, setting hint to benchmark_app.py will report error as in description.